### PR TITLE
DB-8118 SYNONYM tables don't show "describe" information correctly

### PR DESCRIPTION
**Pull request by changli6**
The fix was in ij, so no testcase / ITs were added. Testing steps were in [DB-8118](https://splicemachine.atlassian.net/browse/DB-8118)

### CODE REVIEW

File: db-tools-ij/src/main/grammar/ij.jj [hunk 1]
```diff
@@ -1175,6 +1175,29 @@ class ij {
...
 
 			DatabaseMetaData dbmd = theConnection.getMetaData();
+
+            //Check if it's a synonym table
+            String getSynonymAliasInfo = "SELECT ALIASINFO \n" +
+                    " FROM \n" +
+                    " SYS.SYSALIASES A, SYS.SYSTABLES T, SYSVW.SYSSCHEMASVIEW S \n" +
+                    " WHERE A.ALIASTYPE LIKE 'S' AND \n" +
+                    " S.SCHEMAID = T.SCHEMAID AND \n" +
+                    " A.ALIAS = T.TABLENAME AND \n" +
```
Comment (yxia92) [inline db-tools-ij/src/main/grammar/ij.jj]: We need to add A.schemaid = T.schemaid to differentiate aliases of same name in different schemas.

File: db-tools-ij/src/main/grammar/ij.jj [hunk 2]
```diff
@@ -1175,6 +1175,29 @@ class ij {
 			verifyTableExists(schema,table);
 
 			DatabaseMetaData dbmd = theConnection.getMetaData();
+
+            //Check if it's a synonym table
+            String getSynonymAliasInfo = "SELECT ALIASINFO \n" +
+                    " FROM \n" +
+                    " SYS.SYSALIASES A, SYS.SYSTABLES T, SYSVW.SYSSCHEMASVIEW S \n" +
+                    " WHERE A.ALIASTYPE LIKE 'S' AND \n" +
```
Comment (yxia92) [inline db-tools-ij/src/main/grammar/ij.jj]: Using A.aliastype='S' seems better than like here.

File: db-tools-ij/src/main/grammar/ij.jj (hunk 1 above)
Comment (changli6) [reply db-tools-ij/src/main/grammar/ij.jj]: Low level mistake ...

File: db-tools-ij/src/main/grammar/ij.jj (hunk 2 above)
Comment (changli6) [reply db-tools-ij/src/main/grammar/ij.jj]: Better

**Pull request merged**